edweiss-app/utils/memento/utilsFunctions.tsx

```diff
@@ -18,31 +39,65 @@ export const getStatusColor = (status: string) => {
     }
 };
 
-export const handlePress = (card: Memento.Card, selectionMode: boolean, goToPath: () => void, toggleSelection: (card: Memento.Card) => void) => {
-    if (!selectionMode) {
-        goToPath();
-    } else {
-        toggleSelection(card); // Select or deselect
-    }
+/**
+ * mementoStatusIconMap
+ * 
+ * @param {Memento.LearningStatus} - Learning status
+ * 
+ * @returns {IconType} - Icon type of the status
+ */
+export const mementoStatusIconMap: Record<Memento.LearningStatus, IconType> = {
+    "Not yet": "alert-circle",
+    "Got it": "checkmark-done-circle"
 };
 
-// Move this function outside the CardListScreen component
-export const handleCardPress = (
-    index: number,
-    selectionMode: boolean,
-    selectedCardIndex: number | null,
-    setSelectedCardIndex: (index: number | null) => void,
-    modalRef: React.RefObject<BottomSheetModal> // Update ModalType to your modal's type
+/**
+ * mementoStatusColorMap
+ * 
+ * @param {Memento.LearningStatus} - Learning status
+ * 
+ * @returns {Color} - Color of the status
+ */
+export const mementoStatusColorMap: Record<Memento.LearningStatus, Color> = {
+    "Not yet": "red",
+    "Got it": "green"
+};
+
+/**
+ * 
+ * @param {boolean} isDupplicate - Check if the question is duplicated
+ * @param {boolean} isEmptyField - Check if the question field is empty
+ * @param {React.Dispatch<React.SetStateAction<boolean>>} setExistedQuestion - Set the state of the question existence
+ * @param {React.Dispatch<React.SetStateAction<boolean>>} setEmptyField - Set the state of the empty field
+ * @param {React.Dispatch<React.SetStateAction<boolean>>} setIsLoading - Set the state of the loading status
+ * 
+ * @returns {number} - 0 if the question is duplicated or the field is empty, 1 otherwise
+ */
+export const checkDupplication_EmptyField = (
```

Comment:
Why return 0 or 1 instead of a boolean? Do you plan on returning other values in the future?

Reply:
Hm ... I forgot that we can return a boolean. But I guess in the future, if there is some edge cases, there might be exception so it will return different values for each cases.

Reply:
Still hung up on C, I see.. (jk lol)

Reply:
By the way, if you need to return more than 2 types, consider using a custom type, i.e returnType = "valid"|"error_one"|"error_two". The intention is clearer than when you use a number.

Reply:
Oh nice idea. Thank you :D